examples/live/check_results.py

```diff
@@ -6,203 +6,230 @@
             
-            if print_template_parameters:
-                  log.info('IFO End Times: '+str(end_time)) 
-                  log.info('Mass 1: %f', mass1)
-                  log.info('Mass 2: %f', mass2)
-                  log.info('Spin1z: %f', spin1z)
-                  log.info('Spin2z: %f', spin2z)
-            
-            #check if trigger properties match injection's properties
-            for t in end_time:
-                  if not close(t,inj_time,1.0):    
-                        coinc_fail = True
-                        log.error('Trigger time does not match injection time')  
-            if not close(mass1, inj_mass1, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger mass 1 does not match injection mass 1')
-            if not close(mass2, inj_mass2, 5e-7):
-                  coinc_fail = True                  
-                  log.error('Trigger mass 2 does not match injection mass 2')
-            if not close(spin1z, inj_spin1z, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger spin1z does not match injection spin1z')
-            if not close(spin2z, inj_spin2z, 5e-7):
-                  coinc_fail = True
-                  log.error('Trigger spin2z does not match injection spin1z')   
-            if not close(network_snr, inj_snr, 0.1):
-                  coinc_fail = True
-                  log.error('Network SNR test failed')
+        
+        log.info('IFO SNRs: '+str(snr_list))
+        log.info('Network SNR: %f', network_snr)
+        log.info('IFO End Time: %f', trig_props['tc'][x]) 
+        log.info('Mass 1: %f', trig_props['mass1'][x])
+        log.info('Mass 2: %f', trig_props['mass2'][x])
+        log.info('Spin1z: %f', trig_props['spin1z'][x])
+        log.info('Spin2z: %f', trig_props['spin2z'][x])
+     
+
+    # check if injections match trigger params
+    for i in range(n_injs):
+        has_match = False
+        for j in range(n_coincs):
+            if close(inj_time[i],trig_props['tc'][j],1.0):
+                match = True
+                if not close(inj_mass1[i], trig_props['mass1'][j], 5e-7):
+                    match = False
+                    log.error("Injection %i mass1 does not match simultaneous trigger's mass1",i)
+                if not close(inj_mass2[i], trig_props['mass2'][j], 5e-7):
+                    match = False
+                    log.error("Injection %i mass2 does not match simultaneous trigger's mass2",i)
+                if not close(inj_spin1z[i], trig_props['spin1z'][j], 5e-7):
+                    match = False
+                    log.error("Injection %i spin1z does not match simultaneous trigger's spin1z",i)
+                if not close(inj_spin2z[i], trig_props['spin2z'][j], 5e-7):
+                    match = False
+                    log.error("Injection %i spin2z does not match simultaneous trigger's spin2z",i)
+                if not close(15.0, trig_props['net_snr'][j], 0.05):
```

Comment:
I suggest making the tolerance 1 here, as we do not generally expect the SNR to be stable to a better precision than that.